spinta/dimensions/param/helpers.py

```diff
@@ -7,17 +7,35 @@
 from spinta.dimensions.param.components import ParamLoader
 from spinta.manifests.components import Manifest
 from spinta.nodes import load_node
+from spinta.ufuncs.loadbuilder.components import LoadBuilder
 from spinta.utils.schema import NA
 
 
+def load_param_formulas(context: Context, param: Param, prepare_asts: list[dict]) -> None:
+    formulas = []
+    builder = LoadBuilder(context)
+    for source, ast in zip(param.sources, prepare_asts):
+        builder.update(this=source, param=param)
+
+        # If builder cannot resolve formula - save it in formula and leave resolving for later
+        expr = asttoexpr(ast)
+        if isinstance(expr, Expr):
+            if formula := builder.resolve(expr):
```

Comment:
This could be done, since the second part (after `and`) won't be evaluated if the first part fails:
```suggestion
        if isinstance(expr, Expr) and (formula := builder.resolve(expr)):
```